CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [7.0.0](https://github.com/puppeteer/puppeteer/compare/v6.0.0...v7.0.0) (2021-02-02)
+
+
+### ⚠ BREAKING CHANGES
+
+* **chromium:** - `page.screenshot` cuts screenshot content by the ViewPort size, not ViewPort position.
```

Comment:
```suggestion
* **chromium:** - `page.screenshot` makes a screenshot with the clip dimensions, not cutting it by the ViewPort size.
```

Reply:
The changelog is automatically populated based on the commit messages. If we make changes now they will be overwritten later.